command/agent_info.go

```diff
@@ -53,14 +54,26 @@ func (c *AgentInfoCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Sort and output agent info
 	var stats map[string]interface{}
 	stats, _ = info["stats"]
+	stats_keys := make([]string, 0, len(stats))
+	for key := range stats {
+		stats_keys = append(stats_keys, key)
+	}
+	sort.Strings(stats_keys)
+
+	for _, key := range stats_keys {
+		c.Ui.Output(key)
+		d, _ := stats[key].(map[string]interface{})
```

Comment:
Is there a better variable name for `d` here? It used to be short for `data` but this was replaced. Between the inner loop and the addition of `_keys` I'm having a hard time following which variable holds what data. I'm curious if we can name these a bit better like "stat_name" or "agent_info" or something.